docs/products/access/topics/oidc/claims-lifecycle/claims-lifecycle.md

```diff
@@ -0,0 +1,76 @@
+# User claims and attributes lifecycle
+
+This section aims to explain and visualise how Access deals with user claims/attributes during the most commonly used flows.
+
+## Authorization
+
+![AWS Configuration](img/access-claims-lifecycle-authorization.jpg)
+
+1. The OAuth Client / OIDC Relying Party initiates the authorisation code flow.
+2. Based on the _client_id_ parameter (optionally also _idp_id_), Access resolves the IdentityProvider instance that will be used for user
+   authentication.
+    1. If the underlying IdentityProvider is of type OIDC (e.g. Tulip), Access redirects the user to the defined OIDC Provider with another
+       authorization request.
+        1. Once authenticated, the OIDC Provider redirects back to Access with the authorisation code and Access exchanges the code for the
+           AccessToken / IDToken pair. The IDToken is the primary source of information about the user. An internal User Object is created
+           and populated with all the user claims available in the IDToken. In addition to the user attributes, the AMR claim is also stored
+           in the User Object.
+        2. If the IdentityProvider is of type Tulip, it is likely that a User Data Enhancer (UDH) API integration is configured. If this is
+           the case, the user claims are requested via another API call from UDH, Access passes the user identifier (subject) and a set of
+           scopes requested by the client. Otherwise, the User Object is persisted (see #2.a.5).
+        3. The UDH response is mapped to a UserInfo standard format.
+        4. The User Object is populated with any claims returned by the UDH service. If a given claim already exists in the user object,
+           it's overwritten with the one coming from UDH, which is considered more recent.
+        5. The User Object with all the claims is persisted.
+    2. If the underlying IdentityProvider is of SAML type (e.g. CIM), Access redirects the user to the defined SAML Identity Provider with
+       SAML AuthnRequest (whether it is a redirect or a POST call depends on the integration characteristic).
+        1. Access processes the SAML AuthnResponse.
+           SAML [Attributes Mapping](https://docs.onegini.com/products/access/topics/general-app-config/identity-providers/identity-providers/#attribute-mapping)
+           is triggered if any are defined for a given SAML Identity Provider. If no mappings are available all SAML assertion attributes
+           are used in the next step.
+        2. SAML assertion attributes are mapped to claims according to the configuration. Any unmapped attributes are discarded/filtered
+           out.
+        3. An internal user object is created and populated with all user claims provided by the SAML assertion. Next to the user
+           attributes, the ARR (AMR) assertion is also stored in the User Object.
```

Comment:
if ARR is a typo, maybe we shouldn't mention it in public docs, and just call it AMR

Reply:
And we can mention ACR is also stored (after SAAS-1036 is implemented 😅)

Reply:
Fair point! Removed the ARR 👍